appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -4215,6 +4215,10 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @DefaultMessage("encoding")
   @Description("")
   String encodingParams();
+  
+  @DefaultMessage("decoding")
+  @Description("")
+  String decodingParams();
```

Comment:
This change is incorrect and not even related to your new feature 😄 , please remove this and create a following Ode message:
```java
@DefaultMessage("UriDecode")
@Description("")
String UriDecodeMethods();
```